obs-studio-server/source/nodeobs_service.cpp

```diff
@@ -2271,17 +2279,14 @@ void OBS_service::duplicate_encoder(obs_encoder_t** dst, obs_encoder_t* src, uin
 	}
 }
 
-void OBS_service::releaseStreamingOutput()
+void OBS_service::releaseOutputWithActiveDelay(obs_output_t* output)
 {
-	if (config_get_bool(ConfigManager::getInstance().getBasic(), "Output", "DelayEnable")) {
-		uint32_t delay = obs_output_get_active_delay(streamingOutput);
-		while (delay != 0) {
-			delay = obs_output_get_active_delay(streamingOutput);
-		}
+	uint32_t delay = obs_output_get_active_delay(output);
```

Comment:
I would add a check to at least make sure the output is non-null.

Reply:
Done